lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -97,9 +97,7 @@ <h3 class="hd hd-3">${_("Reports Available for Download")}</h3>
 
     ## Translators: a table of URL links to report files appears after this sentence.
     <p>
-        ${Text(_("{note_start}: To keep student data secure, you cannot save or email these links for direct access. Copies of links expire within 5 minutes.")).format(
-            note_start=HTML("<strong>Note</strong>")
-        )}
+        ${_("<strong>Note</strong>: To keep student data secure, you cannot save or email these links for direct access. Copies of links expire within 5 minutes.")}
```

Comment:
@robrap Is this OK, given I added `expression_filter="h"` to the top of the file? It wasn't working the other way. Same with the below file.